onward/app/models/MostPopularNx2.scala

```diff
@@ -56,6 +56,21 @@ object OnwardItemNx2 {
     }
 
   }
+
+  // We ideally want this to be replaced by something else in the near future. Probably
+  // image-rendering or similar. But this will do for now.
+  // TODO: Replace this.
+
+  def getImageSources(imageMedia: Option[ImageMedia]): Map[String, Option[String]] = {
+    var images = for {
+      profile: ImageProfile <- List(Item300, Item460)
+      width: Int <- profile.width
+      trailPicture: ImageMedia <- imageMedia
+    } yield{
+      width.toString -> profile.bestSrcFor(trailPicture)
+    }
+    images.toMap
+  }
```

Comment:
reason I mentioned scalafmt is because I didn't know whether it'd pick up on the missing newline between this brace and the next `def`